hooks/pre-commit

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+DOCKER_REGISTRY=quay.io
+
+docker pull ${DOCKER_REGISTRY}/typeform/gitleaks-config
```

Comment:
Should we try to detect if docker is installed before this?

Reply:
Done in fbd44fe